src/components/App/App.js

```diff
@@ -5,62 +5,61 @@ import MoviePage from '../MoviePage/MoviePage';
 import './App.css';
 // import movieData from '../../data/movieData';
 
-
 class App extends React.Component {
-  constructor()  {
-    super()
-      this.state = {
-        movies: [],
-        isLoading: true,
-        currentMovie: "",
-        test: true,
-        error: false
-      }
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      isLoading: true,
+      currentMovie: '',
+      test: true,
+      error: false,
+    };
   }
 
   componentDidMount = () => {
     const url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/';
-    const url2 = 'https://httpstat.us/500'
     fetch(url)
-    .then(this.checkForError)
-    .then(data => this.setState({movies : data, isLoading : false}))
-    .catch(error => console.log('Something went wrong:', error))
-  }
+      .then(this.checkForError)
+      .then((data) => this.setState({ movies: data, isLoading: false }))
+      .catch((error) => console.log('Something went wrong:', error));
+  };
 
   checkForError = (response) => {
-    console.log(response)
+    console.log(response);
     if (!response.ok) {
       const statusCode = response.status;
-      this.setState({ error: true});
-      throw new Error(`Something went wrong, please try again. Error Code: ${statusCode}`)
+      this.setState({ error: true });
```

Comment:
This is similar to the `isLoading` feedback above. I wonder if storing the actual error as a string in state would be more valuable than just a boolean.